src/ExamplesGenerator/AsciiDocGenerator.cs

```diff
@@ -65,6 +76,9 @@ public static void GenerateExampleAsciiDoc(IEnumerable<ImplementedExample> examp
 					.AppendLine("If you wish to submit a PR to change this example, please change the source method above")
 					.AppendLine("and run dotnet run -- asciidoc in the ExamplesGenerator project directory.")
 					.AppendLine("////")
+					.AppendLine()
+					.AppendLine($"// {GetReferencePage(example)}.asciidoc:{example.StartLineNumber}")
```

Comment:
Think this should be on the first line of the file, based on the proposal?

Reply:
`{example.StartLineNumber}` refers to the startline of the C# method in the file and not the line number of the console example in the alternative report.

Reply:
OK, I will replace with the alternate line number and push to the top of the file.